build.rs

```diff
@@ -6,7 +6,8 @@ use clap::Shell;
 include!("src/app.rs");
 
 fn main() {
-    let outdir = match std::env::var_os("OUT_DIR") {
+    let var = std::env::var_os("OVRD_OUT_DIR").or_else(|| std::env::var_os("OUT_DIR"));
```

Comment:
Thank you very much!

What about naming this something like `SHELL_COMPLETIONS_DIR`?

Also, I think you can use `Option::or` here to avoid the lambda expression.